docs/usage/self-hosted-configuration.md

```diff
@@ -199,7 +199,7 @@ You probably have no need for this option - it is an experimental setting for th
 You can customize the Git author that's used whenever Renovate creates a commit.
 The `gitAuthor` option accepts a RFC5322-compliant string.
 
-If you are migrating from a old Git author to a new Git author, put the old `gitAuthor` into `RENOVATE_LEGACY_GIT_AUTHOR_EMAIL` in environment (TODO: Explain where/what is this "environment"???).
+If you are migrating from a old Git author to a new Git author, put the old `gitAuthor` into `RENOVATE_LEGACY_GIT_AUTHOR_EMAIL` in environment.
```

Comment:
To me it's not clear what this "environment" is referring to? I guess "environment variable"?

Reply:
Actually this should be in the new https://docs.renovatebot.com/configuration-options/#gitignoredauthors

Reply:
Will move this. 😄